requirements.txt

```diff
@@ -1,7 +1,2 @@
-inflection==0.3.1
-simplejson==3.8.2
-Twisted==19.7.0
-pytz==2016.7
-colorama==0.3.3
-pygments==2.2.0
-ruamel.yaml==0.16.0
+-r requirements/dev.txt
+-r requirements/common.txt
```

Comment:
now the `./install` script which is used in production will l dev requirements 👎 
You can keep this file for convenience (rename: `requirements_all.txt` for example)
but update `install` script to:
- by default install only common requirments
- support a flag --dev that will install the dev requirements as well
- use `--dev` in makefile